src/did/did_doc.rs

```diff
@@ -47,17 +45,32 @@ pub enum VerificationMethodType {
     EcdsaSecp256k1VerificationKey2019,
     X25519KeyAgreementKey2020,
     Ed25519VerificationKey2020,
-    Other(String),
+    Other,
```

Comment:
Do we need it these changes?

Reply:
Yes, we need them to support DIDDoc and Secret types in uniffi bindings. Unfortunately, uniffi doesn't support tuple structs in Enums, but algebraic enums with explicit variables work fine.